client/modules/module_manager.js

```diff
@@ -112,7 +112,7 @@ define(function(require) {
     // Server has asked us to load a new module.
     network.on('loadModule', function(bits) {
       var def = bits.module;
-      var code = bits.def;
+      var code = bits.module.def;
```

Comment:
Can we do anything about the variable names here so the server / client match up a little better?